src/fsharp/Fsc/Fsc.fsproj

```diff
@@ -23,16 +22,28 @@
     <AllowCrossTargeting>true</AllowCrossTargeting>
     <OtherFlags>$(OtherFlags) --warnon:1182</OtherFlags>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
```

Comment:
I believe this project file needs reverting, see the project.json changes below. I trust @dsyme can help you integrate some compiler unit tests

Reply:
Oops, didn't mean to commit this. Good spot